python/core/auto_generated/processing/qgsprocessingparameters.sip.in

```diff
@@ -2887,9 +2887,16 @@ An expression parameter for processing algorithms.
 %End
   public:
 
+    enum Type
```

Comment:
could we move this to an enum class in Qgis? E.g Qgis::ExpressionType ?

Reply:
Done